{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/models/__init__.py

```diff
@@ -62,6 +62,9 @@ def includeme(config):
     # use pyramid_tm to hook the transaction lifecycle to the request
     config.include('pyramid_tm')
 
+    # use pyramid_retry to allow retrying N times before indicating failure
```

Comment:
"use pyramid_retry to retry a request when transient exceptions occur" should be sufficient